kratos/utilities/properties_utilities.cpp

```diff
@@ -0,0 +1,46 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/properties.h"
+#include "utilities/properties_utilities.h"
+
+namespace Kratos
+{
+namespace PropertiesUtilities
+{
+void CopyPropertiesValues(
+    const Properties& rOriginProperties,
+    Properties& rDestinatiionProperties
+    )
+{
+    KRATOS_TRY
+
+    const auto& r_origin_data = rOriginProperties.Data();
```

Comment:
@pooyan-dadvand can u take a look? this looks correct to me but i am not 100% sure.

@loumalouomega if you can try to pass valgrind on the test to see if it is clean

Reply:
Do you want me to post the output here?

Reply:
It is ok!